app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -18,8 +19,13 @@ def create
   end
 
   def destroy
+    if session[:signing_out_for_inactivity]
+      flash_message = { notice: I18n.t('messages.access.signed_out_for_inactivity') }
+    else
+      flash_message = { success: I18n.t('messages.access.signed_out') }
```

Comment:
Check with UX, but I think this should be `notice` for both

Reply:
Correct https://ukgovernmentdfe.slack.com/archives/CG9A8H1HP/p1589793769222100?thread_ts=1589793583.221500&cid=CG9A8H1HP